### PR TITLE
hide tooltips on hide column

### PR DESCRIPTION
**Pull request by hwrainbird**
- Sets tooltip icon to display: none when column is hidden
- Hides tooltip icon when column is collapsed

### CODE REVIEW

File: KTL.js [hunk 1]
```diff
@@ -11398,6 +11398,12 @@ function Ktl($, appInfo) {
                     .find('.table-fixed-label')
                     .css('display', 'inline-block');
 
+                // Hide the tooltip icon if it's in a collapsed column
```
Comment (CharlesBrunelleIO) [inline KTL.js]: This comment doesn't add information that the code isn't already implying

File: KTL.js [hunk 2]
```diff
@@ -11487,6 +11493,7 @@ function Ktl($, appInfo) {
                 }
 
                 function hideColumn(viewId, columnIndex) {
+                    $(`#${viewId} tr > th:nth-child(${columnIndex}) i.ktlTooltipIcon`).hide(); // Hide tooltip icons
```
Comment (CharlesBrunelleIO) [inline KTL.js]: Same with this trailing comment. Write a comment to add information that is not in the code. A "why" for example.

File: KTL.js (hunk 1 above)
Comment (cortexrd) [reply KTL.js]: True.  Only add comments to explain things that are not obvious.  Your function and variable names should be explicit enough so they don't require comments.

File: KTL.js (hunk 2 above)
Comment (hwrainbird) [reply KTL.js]: Okay noted, thanks for advice

**Review by hwrainbird — COMMENTED** [review]
Removed comments

**Review by cortexrd — APPROVED** [review]
Perfect, thank you!

**Pull request merged**